### PR TITLE
Fix disable usage options

### PR DESCRIPTION
**Pull request by MouazAlzahabi**
Handles #2825

### CODE REVIEW

File: Multisig/Features/Security/KeychainItem.swift [hunk 1]
```diff
@@ -110,6 +110,9 @@ enum KeychainItem {
             ]
             if let context = LAContext(password: password) {
                 privateKeyAttrs[kSecUseAuthenticationContext] = context
+            } else {
+                // In case password is worng
```
Comment (elgatovital) [inline Multisig/Features/Security/KeychainItem.swift]: I would also add short explanation here regarding case when device password is not set (failure to create LAContext)

File: Multisig/Features/Security/SecurityCenter.swift [hunk 2]
```diff
@@ -148,9 +148,20 @@ class SecurityCenter {
 
             AppSettings.securityLockMethod = newMethod
 
-            try changeStoreSettings(currentPlaintextPassword: oldPasscode, newPlaintextPassword: changedPasscode, store: sensitiveStore)
-            try changeStoreSettings(currentPlaintextPassword: oldPasscode, newPlaintextPassword: changedPasscode, store: dataStore)
-            NotificationCenter.default.post(name: .securitySettingsChanged, object: self)
+            // Change settings should be done only for the enabled store
+            // Change settings for disabled store will through an exception
```
Comment (elgatovital) [inline Multisig/Features/Security/SecurityCenter.swift]: typo in comment: will throw an exception

File: Multisig/Features/Security/KeychainItem.swift [hunk 3]
```diff
@@ -68,9 +68,6 @@ enum KeychainItem {
 
             if let context = LAContext(password: password) {
                 result[kSecUseAuthenticationContext] = context
-            }  else {
```
Comment (elgatovital) [inline Multisig/Features/Security/KeychainItem.swift]: what if keychain item has been initialised when device password was not set? Wouldn't this edge-case lead to problems?

**Review by DmitryBespalov — APPROVED** [review]

File: Multisig/Features/Security/KeychainItem.swift (hunk 3 above)
Comment (MouazAlzahabi) [reply Multisig/Features/Security/KeychainItem.swift]: This we will do in next step

**Pull request merged**